api/src/main/java/org/apache/iceberg/metrics/LoggingMetricsReporter.java

```diff
@@ -28,6 +28,11 @@
  */
 public class LoggingMetricsReporter implements MetricsReporter {
   private static final Logger LOG = LoggerFactory.getLogger(LoggingMetricsReporter.class);
+  private static final LoggingMetricsReporter INSTANCE = new LoggingMetricsReporter();
```

Comment:
This is small enough I don't think it's a problem.

Reply:
it's in 2 PRs because I thought that the other PR would have been merged by now. Also I thought it's a small enough diff that wouldn't cause any problems for either of the 2 PRs